tutorials/setting-up-redis.md

```diff
@@ -0,0 +1,250 @@
+1. Update the Debian packages list. In the SSH terminal, enter the following command:
+
+        sudo apt-get update
+
+1. Install Redis:
+
+        sudo apt-get -y install redis-server
+
+1. Verify that Redis is running:
+
+        ps -f -u redis
+
+   You should see something like:
+
+        yourname@redis-tutorial:~$ ps -f -u redis
+        UID        PID  PPID  C STIME TTY          TIME CMD
+        redis      802     1  0 21:07 ?        00:00:00 /usr/bin/redis-server 127.0.0.1:6379
+
+Congratulations, Redis is running, but will only accept connections from `127.0.0.1` - the local machine
+running the Redis server.
+
+## Configure Redis remote access
+
+By default, Redis doesn't allow remote connections. To change this setting,
+you can change the configuration in the [`redis.conf`](https://redis.io/topics/config) file.
+Follow these steps.
+
+### Edit `redis.conf`
+
+1. In the SSH terminal window, edit `redis.conf`. This tutorial uses the
+`nano` editor, but you can substitute your favorite editor. Enter:
+
+        sudo nano /etc/redis/redis.conf
+
+1. Scroll down to the line that begins with `bind 127.0.0.1`
+
+1. Replace `127.0.0.1` with `0.0.0.0`:
+
+        bind 0.0.0.0
+
+   The `bind 0.0.0.0` setting enables Redis to accept connections from any IP address. This is
+   a commonly used setting. When bound to `127.0.0.1`, Redis will only accept connections
+   from the local machine running Redis -- meaning your application would have to reside on the
+   same machine as the Redis server.
+
+1. Save the file and exit the editor.
+
+1. Restart the database service. In the SSH terminal, enter:
+
+        sudo service redis-server restart
+
+1. Verify that Redis is listening to all traffic:
+
+        ps -f -u redis
+
+   You should now see something like:
+
+        yourname@redis-tutorial:~$ ps -f -u redis
+        UID        PID  PPID  C STIME TTY          TIME CMD
+        redis      950     1  0 21:07 ?        00:00:00 /usr/bin/redis-server 0.0.0.0:6379
+
+Congratulations, Redis will now accept connections from external machines.
+
+## Open the network port (optional)
+
+By default, this Compute Engine instance is added to the `default` network in your project.
+The `default` network allows all TCP connections between its Compute Engine instances using
+the internal network. In a production environment, you will want to skip this step.
+
+Redis accepts remote connections on TCP port 6379. Follow these steps to add
+a firewall rule that enables traffic on this port.
+
+
+1. In the Cloud Console, navigate to the **Create a firewall rule**
+   page.
+
+    **[Create a firewall rule](https://console.cloud.google.com/networking/firewalls/add)**
+
+1. In the **Network** field, leave the network as **default**.
+
+1. In the **Name** field, enter:
+
+        redis-tutorial
+
+1. In a separate window, navigate to [ip4.me](http://ip4.me) to get the IPv4 address of your local computer.
```

Comment:
As a note, this site is used in the setting-up-postgres tutorial. I tried just using Google search, but if you have an ipv6 interface, Google will use ipv6 and it will only have the ipv6 address information. This site only has a ipv4 interface.

The firewall configuration appears to only allow ipv4 rules at this time.